### PR TITLE
Increase minimum PyTorch requirements to 1.2.0

### PR DESCRIPTION
**Pull request by Jean-Francois-Lafleche**
### Description
- Increase minimum PyTorch requirements to 1.2.0
- Add minimum requirement for CUDA to 10.0.130
- Add minimum display driver version to 410.48 (minimum version for CUDA 10.0.130)
- Add minimum PyTorch version assert in setup.py

### Related Issues
#65 

#### PyTorch 0.4.1, CUDA 9
- Docker Container: `pytorch/pytorch:0.4.1-cuda9-cudnn7-devel`
    - python setup.py develop (`fatal error: torch/extension.h: No such file or directory`)
#### PyTorch 1.0, CUDA 10.0
- Docker Container: `pytorch/pytorch:1.0-cuda10.0-cudnn7-devel`
    - python setup.py develop (`error: 'TORCH_CHECK' was not declared in this scope`)
#### PyTorch 1.1.0, CUDA 10.0
- Docker Container: `pytorch/pytorch:1.1.0-cuda10.0-cudnn7.5-devel`
    - python setup.py develop (`error: 'TORCH_CHECK' was not declared in this scope`)
#### PyTorch 1.2, CUDA 9.2
- Docker Container: `nvidia/cuda:9.2-base-ubuntu16.04`
    - pip install torch==1.2.0+cu92 torchvision==0.4.0+cu92 -f https://download.pytorch.org/whl/torch_stable.html
    - python setup.py develop (`error: command '/usr/local/cuda/bin/nvcc' failed with exit status 1`)
#### PyTorch 1.2, CUDA 10.0
- Docker Container: `pytorch/pytorch:1.2-cuda10.0-cudnn7-devel`
    - python setup.py develop (successful build)

### Tests (after changes)
- Docker Container: `pytorch/pytorch:1.2-cuda10.0-cudnn7-devel`
    - python setup.py develop (successful build)
- Docker Container: `pytorch/pytorch:1.0.1-cuda10.0-cudnn7-devel`
    - python setup.py develop (Warning logged)

### CODE REVIEW

**Comment by Caenorst** [comment]
Why are we testing `pytorch/pytorch:1.0.1-cuda10.0-cudnn7-devel` if Pytorch 1.2 is the new minimum requirements ?

**Comment by Jean-Francois-Lafleche** [comment]
@Caenorst  Testing `pytorch/pytorch:1.0.1-cuda10.0-cudnn7-devel` is to confirm that setup will fail with an assert error message if pytorch < 1.2 is installed.

**Comment by Caenorst** [comment]
Do we actually want to have an assertion ? Why not just having a warning message ? I agree that we don't wanna officially support old version of Pytorch / CUDA / python but should we explicitly make the installation crash while it could potentially works ?

**Comment by Jean-Francois-Lafleche** [comment]
I agree. A warning is better than crashing the installation. Fixed.

File: setup.py [hunk 1]
```diff
@@ -28,9 +30,14 @@
...
-# Check that PyTorch version installed meets minimum requirements
-assert torch.__version__ >= '1.2.0', f'Kaolin requires PyTorch >= 1.2.0. Found version {torch.__version__} instead.'
+logger = logging.getLogger()
+logging.basicConfig(format='%(levelname)s - %(message)s')
+
 
+# Check that PyTorch version installed meets minimum requirements
+if torch.__version__ < '1.2.0':
+    logger.warning(f'Kaolin is tested with PyTorch >= 1.2.0. Found version {torch.__version__} instead.')
+    # warnings.warn(f'Kaolin is tested with PyTorch >= 1.2.0. Found version {torch.__version__} instead.')
```
Comment (Caenorst) [inline setup.py]: Please remove commented out code

**Review by Caenorst — APPROVED** [review]
just a little comment to remove and we are good to go. Thanks !

File: setup.py (hunk 1 above)
Comment (Jean-Francois-Lafleche) [reply setup.py]: Good catch. Fixed

**Pull request merged**